### DISCUSSION TITLE
How to open a new tab by command 'open'. (Mac)

### DISCUSSION THREAD

**Discussion by xk12138** [Eugeny/tabby#5781] [Q&A] [answered]
Here, I want to use command 'open' to open this Tabby.app. But there is no available tab. Is there any args to open a new tab?

**Comment by Eugeny** [comment, marked as answer]
Check out `/Applications/Tabby.app/Contents/MacOS/Tabby --help`. 

* `/Applications/Tabby.app/Contents/MacOS/Tabby open <path>` to open a tab in a folder
* `/Applications/Tabby.app/Contents/MacOS/Tabby run <command>` to run a command in a new tab

**Reply by xk12138** [reply]
Thank you!
